### PR TITLE
feat(DENG-1358): updated existing Airflow ETL to use the new Fivetran dataset (new connector)

### PR DESCRIPTION
**Pull request by kik-kik**
# feat(DENG-1358): updated existing Airflow ETL to use the new Fivetran dataset (new connector)

Also, removed `init.sql` files as they are redundant and serve no purpose in this case.

follow-up: https://github.com/mozilla/bigquery-etl/pull/4173

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-1374)

### CODE REVIEW

**Review by ANich — APPROVED** [review]
LGTM if you've tested this.

File: sql/moz-fx-data-shared-prod/monitoring_derived/airflow_dag_run_v1/query.sql
[the remark this answers is not in the document]
Comment (kik-kik) [reply sql/moz-fx-data-shared-prod/monitoring_derived/airflow_dag_run_v1/query.sql:1]: https://mozilla-hub.atlassian.net/browse/DENG-1351

[the remark this answers is not in the document]
Comment (kik-kik) [reply sql/moz-fx-data-shared-prod/monitoring_derived/airflow_dag_run_v1/query.sql:1]: In short: after the Airflow migration the DB changes and we had to update the Fivetran connector which affected the tables where the data lands.

File: sql/moz-fx-data-shared-prod/monitoring_derived/airflow_dag_run_v1/query.sql [hunk 1]
```diff
@@ -7,6 +7,6 @@ SELECT
   start_date,
   end_date,
 FROM
-  `moz-fx-data-bq-fivetran.airflow_metadata_airflow_db.dag_run`
+  `moz-fx-data-bq-fivetran.telemetry_airflow_metadata_public.dag_run`
```
Comment (sean-rose) [inline sql/moz-fx-data-shared-prod/monitoring_derived/airflow_dag_run_v1/query.sql:10]: I don't think they kept any historical data in the migration, right?  So just using the new database in all these ETLs means we lose all history prior to the migration.  Should we include both the current and historical data instead?

Comment (lelilia) [reply sql/moz-fx-data-shared-prod/monitoring_derived/airflow_dag_run_v1/query.sql:10]: hmmm good point ... it's mainly for an airflow monitoring dashboard so not sure how much value old stuff has ... but it couldn't hurt either

Comment (kik-kik) [reply sql/moz-fx-data-shared-prod/monitoring_derived/airflow_dag_run_v1/query.sql:10]: ok, now including data pre-migration for `dag_run`, `task_fail`, and `task_reschedule` tables.

**Review by lelilia — APPROVED** [review]

**Pull request merged**